src/backend/commands/extension.c

```diff
@@ -100,7 +103,7 @@ static List *find_update_path(List *evi_list,
 static void get_available_versions_for_extension(ExtensionControlFile *pcontrol,
 									 Tuplestorestate *tupstore,
 									 TupleDesc tupdesc);
-static void ApplyExtensionUpdates(AlterExtensionStmt *stmt, Oid extensionOid,
+static void ApplyExtensionUpdates(Oid extensionOid,
```

Comment:
This change seems to be ported from another patch. Do we need this?

Reply:
Fixed